src/main/java/com/metamx/common/io/smoosh/FileSmoosher.java

```diff
@@ -65,9 +68,12 @@
 
   private final List<File> outFiles = Lists.newArrayList();
   private final Map<String, Metadata> internalFiles = Maps.newTreeMap();
-
+  private Map<String, File> createdFiles = Maps.newLinkedHashMap();
```

Comment:
a nit, but let's name it `createdTemporaryFiles()` to highlight that these are temporary